exampleSite/layouts/partials/custom_footer.html

```diff
@@ -0,0 +1,3 @@
+<script type="text/javascript">
+	console.log("Custom Footer is on!");
```

Comment:
Do we need this here? 🤔

Reply:
We don't really, it's more of a POC that it works. I can remove them.

Reply:
I removed this file.